<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="checkBox1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox3</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
